src/main/java/wolox/training/services/OpenLibraryService.java

```diff
@@ -0,0 +1,58 @@
+package wolox.training.services;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Service;
+import wolox.training.exceptions.BookNotFoundException;
+import wolox.training.helpers.Json;
+import wolox.training.services.dtos.BookDTO;
+
+@Service("openLibraryService")
+public class OpenLibraryService {
+
+    @Autowired
+    @Qualifier("environment")
+    private Environment environment;
+
+    private static final String OPEN_LIBRARY_PATH = "/api/books";
+
+    private static final String OPEN_LIBRARY_PARAMS = "?bibkeys=ISBN:%s&format=json&jscmd=data";
+
+    public wolox.training.pojos.dtos.BookDTO bookInformation(String isbn)
```

Comment:
Try to avoid duplicate class names so that you don't get conflicts with imports 😺